k8s/overlays/coreso/coreso-databridge-claims/cse-export-d2cc/kustomization.yaml

```diff
@@ -2,5 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - cse-export-d2cc-cgm-data-bridge-pv-claim.yaml
-  - cse-export-d2cc-crac-data-bridge-pv-claim.yaml
+  - cse-export-d2cc-data-bridge-pv-claim.yaml
```

Comment:
General remark for Azure and Coreso, It's better to collect all the pv-claim.yaml file in the same folder coreso-databridge-claims  and azure-databridge-claim witj only one kustomize to decrease the number of files